docs/user-guide/registry/FAQ.md

```diff
@@ -1,94 +1,102 @@
-# 服务注册常见问题
+# Service Registration FAQ
 
-本文主要介绍[服务注册插件](../../../sermant-plugins/sermant-service-registry)在使用时遇到的常见问题。
+[简体中文](FAQ-zh.md) | [English](FAQ.md)
 
-## 报错：No such extension org.apache.dubbo.registry.RegistryFactory by name sc
+This document describes frequently asked questions (FAQs) about using the [service registration plugin](../../../sermant-plugins/sermant-service-registry).
 
-如下图所示：
+## Exception：No such extension org.apache.dubbo.registry.RegistryFactory by name sc
+
+As shown in the following figure:
 
 ![No such extension org.apache.dubbo.registry.RegistryFactory by name sc](../../binary-docs/registry-faq-1.png)
 
-该错误原因有以下3种：
+The possible causes are as follows:
 
-1.1 宿主没有带agent启动。因为原生dubbo并不支持往sc注册，所以如果没带agent启动且配置的注册地址的协议为sc时，就会产生以上报错。
+1.1 The host is not started with an agent. The native dubbo does not support registration with the SC. Therefore, if the agent is not started and the protocol of the registration address is SC, the preceding error is reported.
```

Comment:
maybe is "The application is not started with an agent. The native dubbo does not support registration with the SC. Therefore, if the application is not started with an agent and the protocol of the registration address is SC, the preceding error is reported." ?

Reply:
Resolved